iocore/net/quic/test/test_QUICFrame.cc

```diff
@@ -352,6 +351,8 @@ TEST_CASE("Store STREAM Frame", "[quic]")
     CHECK(memcmp(stream_frame2->data(), "\x11\x22\x33\x44\x55", stream_frame2->data_length()) == 0);
     CHECK(stream_frame2->offset() == 0x100000000 + 5);
     CHECK(stream_frame2->stream_id() == 0x01);
+
+    QUICFrameDeleter::delete_stream_frame(stream_frame2);
```

Comment:
`SECTION("STREAM frame split")` of `TEST_CASE("Retransmit", "[quic][frame][retransmit]")` also have a `split()` call. Should the frame be deleted too?

Reply:
It is stranger that ASAN doesn't report that leaking.

Reply:
Perhaps it has to be deleted, but it wouldn't detected because the memory is allocated with an internal allocator.